frontend/src/components/ReviewPartnerDialog.js

```diff
@@ -53,6 +54,30 @@ const ReviewPartnerDialog = ({
     navigate(homeUrl, { replace: true })
   }
 
+  const renderRating = (icon, label, value, setValue) => {
+    return (
+      <>
+        <Grid item xs={4}>
+          <DialogContentText>
+            <Stack direction="row" alignItems={'center'}>
+              {icon}
+              {label}
+            </Stack>
+          </DialogContentText>
```

Comment:
React throws an error in the console because putting `Stack` inside `DialogContentText` renders a `div` within a `p` tag:
![Screenshot 2022-10-31 at 11 31 42 AM](https://user-images.githubusercontent.com/31149701/198925436-debca314-d293-404d-addb-58c50fd5f646.png)

Maybe we can use `DialogContentText` to only wrap the `{label}`?

Reply:
Oh okay, I will look into this. I wanted to align center because it wasn't really in the center.